content/terraform-plugin-framework/v1.15.x/docs/plugin/framework/resources/identity.mdx

```diff
@@ -0,0 +1,194 @@
+---
+page_title: Identity
+description: >-
+Learn how to implement resource identity in the Terraform plugin framework.
+---
+
+# Resource identity
+
+A resource identity is a data object, determined by the provider, that is stored alongside the resource state to uniquely identify a remote object. A resource identity should have the following properties:
+- The resource identity must correspond to at most one remote object per provider, across all instances of that provider.
+- Given a resource identity (during `import`), the provider must be able to determine whether the corresponding remote object exists, and if so, return the resource state.
+- Unless the resource identity schema is being upgraded, the identity data must not change during the lifecycle of the remote object.
+- That is, from the creation of the remote object in the remote system until its destruction.
```

Comment:
I added this bullet point above because I felt like they were covering the same thing so you can delete this one!
```suggestion
```